src/markdown.rs

```diff
@@ -30,10 +57,22 @@ async fn render_readme(tera: &Tera, data: &Data) -> Result<()> {
     Ok(())
 }
 
-async fn render_projects(github: &Octocrab, tera: &Tera, data: &Data) -> Result<()> {
+async fn render_projects(github: Arc<Octocrab>, tera: Arc<Tera>, data: Arc<Data>) -> Result<()> {
+    let mut tasks = Vec::new();
+    let data = data.clone();
+
     for project in &data.project {
```

Comment:
Maybe we can collect those tasks into [FuturesUnordered](https://docs.rs/futures/latest/futures/stream/futures_unordered/struct.FuturesUnordered.html) and await them.

For example:

```rust
let tasks: FuturesUnordered<_>  = data.project.iter().map(|project|render_project(github, tera, project)).collect();
```

Reply:
Maybe it's better to limit them to eight to avoid sending too many requests simultaneously?

Reply:
I made the update to `FuturesUnordered`. Overall it is slightly slower then the original update, but we are talking milliseconds so no noticeable difference.

For limiting concurrency, is it possible to do that without using a tokio Semaphore and therefore needing to spawn? I don't see a way to do it with `futures`, but I'm also not failure with it.